content/post/051-new-ipfs-release-process.md

```diff
@@ -0,0 +1,209 @@
+---
+date: 2019-07-18
+url: 2019-07-18-ipfs-release-process
+tags: IPFS, go-ipfs, release process, 
+title: Improving the IPFS Release Process
+author: Steven Allen, Alan Shaw, David Dias, Molly Mackinlay
+---
+
+go-ipfs is introducing a new release cycle and process to ensure reliable releases.
+
+## Background
+
+The past three releases have had a few critical regressions, several of which
+would have been caught by better release testing and released faster with an
+improved release process.
```

Comment:
```suggestion
improved release process. At the end of Q2, we took the time to reflect and investigate these past challenges to identify specific strategies to prevent them going forward. This is what we found:
```
I think this intro needs more context and support. The reason we're diving into this background is to give clarity about why our chosen strategy should help us avoid these problems in the future.

Reply:
(section reworked)